styles/colors-cobalt.less

```diff
@@ -75,14 +77,14 @@
 @agate-button-highlighted-color: @agate-button-focus-color;
 @agate-button-highlighted-bg-color: @agate-button-focus-bg-color;
 @agate-button-highlighted-bg-image: none;
-@agate-button-selected-color: @agate-highlight;
-@agate-button-selected-bg-color: @agate-accent;
+@agate-button-selected-color: @agate-selected-color;
+@agate-button-selected-bg-color: @agate-selected-bg-color;
 
 // Checkbox
 // ---------------------------------------
 @agate-checkbox-text-color: @agate-white;
 @agate-checkbox-bg-color: @agate-accent;
-@agate-checkbox-focus-bg-color: @agate-background-focus;
+@agate-checkbox-focus-bg-color: @agate-focus-bg-color;
```

Comment:
Same here, Is there a reason not to update @agate-picker-item-focus-bg-color like Checkbox?